balloon.go

```diff
@@ -0,0 +1,78 @@
+package firecracker
+
+import(
+	models "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
+)
+
+// BalloonBuilder is a builder that will create a balloon used to set up
+// the firecracker microVM.
+type BalloonBuilder struct {
+	balloon models.Balloon
+}
+
+// NewBalloonBuilder will return a new BalloonBuilder with a given amountMB and DeflateOnOom.
+func NewBalloonBuilder(AmountMb int64, DeflateOnOom bool) BalloonBuilder{
```

Comment:
Shouldn't it take BalloonOpt?

Reply:
I think it is still better to have balloon.go separately.

Reply:
Gotcha.